src/main/java/com/cgdpd/library/service/UserService.java

```diff
@@ -0,0 +1,23 @@
+package com.cgdpd.library.service;
+
+import com.cgdpd.library.dto.user.CreateUserRequestDTO;
+import com.cgdpd.library.dto.user.UserDTO;
+import com.cgdpd.library.entity.UserEntity;
+import com.cgdpd.library.mapper.UserMapper;
+import com.cgdpd.library.repository.UserRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class UserService {
```

Comment:
Not necessary in this PR can be added later when we actually need it to do something, please remove